### PR TITLE
dev-db/pgcli: Bump to 3.1.0, add new dependencies

### PR DESCRIPTION
**Pull request by jsmolic**
This PR bumps `dev-db/pgcli` to version 3.1.0 and adds 7 new python packages. Most notable change in pgcli-3.1.0 is switching from humanize to pendulum library (https://github.com/dbcli/pgcli/commit/8f7e31450835bca5d9a8bb4de252efba6f4b7b10) which adds new `dev-python/pendulum` dep.
`dev-python/pendulum` introduces new `dev-python/pytzdata` package, which needs `dev-python/cleo`.

Cleo depends on `dev-python/clikit`, which needs `dev-python/crashtest`, `dev-python/pylev` and `dev-python/pastel` packages.
I haven't managed to get a working test suite for `dev-python/pendulum` right now so tests are restricted but I will work on fixing them in the future.

### CODE REVIEW

File: dev-python/pendulum/pendulum-2.1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+PYTHON_COMPAT=( python3_{6..9} )
+inherit distutils-r1
+
+DESCRIPTION="Drop-in replacement for the standard datetime class"
+HOMEPAGE="https://pendulum.eustace.io/"
```
Comment (juippis) [inline dev-python/pendulum/pendulum-2.1.2.ebuild:11]: We like to have the Github homepage as 2ndary homepage here too.

File: dev-python/clikit/clikit-0.6.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+PYTHON_COMPAT=( python3_{6..9} )
+inherit distutils-r1
+
+DESCRIPTION="Group of utilities to build beautiful and testable command line interfaces."
```
Comment (juippis) [inline dev-python/clikit/clikit-0.6.2.ebuild:10]: We don't write dots at the end of DESCRIPTION.

File: dev-db/pgcli/pgcli-3.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/setproctitle[${PYTHON_USEDEP}]
+	dev-python/sqlparse[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	test? (
+		dev-db/postgresql
+		dev-python/mock[${PYTHON_USEDEP}]
+	)"
+
```
Comment (juippis) [inline dev-db/pgcli/pgcli-3.1.0.ebuild:35]:
```
>>> Test phase: dev-db/pgcli-3.1.0
 * python3_7: running distutils-r1_run_phase python_test
ImportError while loading conftest '/var/tmp/portage/portage/dev-db/pgcli-3.1.0/work/pgcli-3.1.0/tests/conftest.py'.
tests/conftest.py:3: in <module>
    from utils import (
tests/utils.py:4: in <module>
    from pgcli.main import format_output, OutputSettings
../pgcli-3.1.0-python3_7/lib/pgcli/main.py:18: in <module>
    import pendulum
/usr/lib/python3.7/site-packages/pendulum/__init__.py:26: in <module>
    from .date import Date
/usr/lib/python3.7/site-packages/pendulum/date.py:22: in <module>
    from .exceptions import PendulumException
/usr/lib/python3.7/site-packages/pendulum/exceptions.py:1: in <module>
    from .parsing.exceptions import ParserError  # noqa
/usr/lib/python3.7/site-packages/pendulum/parsing/__init__.py:10: in <module>
    from dateutil import parser
E   ModuleNotFoundError: No module named 'dateutil'
 * ERROR: dev-db/pgcli-3.1.0::testworld failed (test phase):
```

**Review by juippis — APPROVED** [review]
0 issues, 3 nags on an 8-commit-PR. Good work here, thanks! I'll update whatever I mentioned about.

**Comment by jsmolic** [comment]
@juippis Thanks :relaxed:

**Pull request closed without merging**